source/features/pr-jump-to-first-non-viewed-file.tsx

```diff
@@ -28,5 +28,8 @@ void features.add({
 	include: [
 		pageDetect.isPRFiles
 	],
+	exclude: [
+		pageDetect.isPRCommit404
```

Comment:
@fregante we need another detection for 
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/789/files/e01aeec9777d9bb12d418bffaf37221d5f268976..a58b37845f1b2660221de019e4ae6c736feedc26

is that a new detection `isPRFiles404` or join it as one?

Reply:
I wonder if `isPRFiles` should automatically exclude 404 pages.

I'm a fan of potentially having to add this `exclude 404` on every feature that requires it. I almost don't care that the lack of it causes errors in the console (it's a literal 404 error page after all)

Reply:
> I'm a fan of potentially having to add this `exclude 404` on every feature that requires it.

lost you on this one.

Reply:
Nm. I just used exclude